e2e/tests/testServer.ts

```diff
@@ -23,20 +23,22 @@ THE SOFTWARE.
 */
 
 import http from 'http';
+import { AddressInfo } from 'net';
 
-export function createTestHttpServer(hostname = '127.0.0.1', port = 8082) {
+export async function createTestHttpServer(port = 0) {
   const server = http.createServer((_req, res) => {
     res.statusCode = 200;
-    res.setHeader('Content-Type', 'text/plain');
-    res.end('Hello Elastic Synthetics Recorder');
+    res.setHeader('Content-Type', 'text/html');
+    res.end(`
+      <button><div>Hello Elastic Synthetics Recorder</div></button>
```

Comment:
Textarea clicks are not tracked by newer version of PW which made the tests hang. RIght now we can use this button as a trigger for the e2e tests.